core/web/router.go

```diff
@@ -306,45 +306,57 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 	userOrEI.GET("/ping", ping.Show)
 }
 
+// This is higher because it serves main.js and any static images
+var staticAssetsRateLimit = int64(50)
+var staticAssetsRateLimitPeriod = 1 * time.Minute
+var indexRateLimit = int64(10)
+var indexRateLimitPeriod = 1 * time.Minute
+
 func guiAssetRoutes(box packr.Box, engine *gin.Engine) {
-	boxList := box.List()
+	// Serve JS files
+	assetsRouter := engine.Group("/assets", rateLimiter(
```

Comment:
Would there be any value in disabling all rate limiting when `CHAINLINK_DEV=true`?

Reply:
I thought about that actually. I think it would be a good idea

Reply:
Great! Adding this also fixed the tests :D

The only question is should I make this configurable in case there is an issue with the rate limits being too low? Or maybe just a config option to disable the rate limits for the UI